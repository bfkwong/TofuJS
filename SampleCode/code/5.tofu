true and true;
true and false; 
false and true; 
false and false; 

true or true; 
true or false; 
false or true;
false or false; 

1 + 2 + 3 - 3 - 2 - 1 * 2 * 3 * 5 / 4 / 2; 

"String" + "String";

1 > 3; 
2 >= 4; 
3 < 54; 
455 <= 4653; 
"1" > "3"; 
"2" >= "4"; 
"3" < "54"; 
"455" <= "4653"; 


print("hello");
print(2);
print(true); 
print(false);
print(undefined);