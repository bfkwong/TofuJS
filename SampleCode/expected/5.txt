hello
2
true
false
undefined
